### ISSUE TITLE
No alt result

### ISSUE THREAD

**Issue by zbigniew73** [coderedcorp/coderedcms#205]
no alt result
img class="card-img-top w-100" src="/media/images/kontakt.2e16d0ba.fill-900x600.jpg" alt="">

**Comment by vsalvino** [comment]
Thanks for the bug report @zbigniew73 . Having a blank `alt` tag is valid HTML and tells the screen reader or browser that this image really is not important and should not have a placeholder, so to speak.

That being said, it looks like this image is appearing in cards somewhere. Do you think it would make sense to use the image's title or some other attribute to fill the alt tag in this instance?

**Comment by vsalvino** [comment]
Closing due to inactivity. Please re-open if any new issues or information arises.
